index.html

```diff
@@ -0,0 +1,141 @@
+          <p>Vorname Nachname<br />Position</p>
+        </div>
+
+        <div class="worker">
+          <img src="images/Koch_quadratisch.png" alt="Banana Bread">
+          <p>Vorname Nachname<br />Position</p>
+        </div>
+
+        <div class="worker">
+          <img src="images/Koch_quadratisch.png" alt="Banana Bread">
+          <p>Vorname Nachname<br />Position</p>
+        </div>
+      </div>
+
+      <div id="team">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+      </div>
+
+      <div id="contact">
+        <h2>Kontakt</h2>
+      </div>
+
+      <div class="box2">
+        <div class="map">
+          <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d10960.292996253223!2d8.0459933!3d46.625315!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x4ed11045e4a7d95b!2sHotel%20Fiescherblick!5e0!3m2!1sde!2sch!4v1665255321505!5m2!1sde!2sch" width="700" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+        </div>
+
+        <div class="openinghours">
+          <h3>Unsere Öffnungszeiten</h3>
+          <p>Di-Sa: 17 Uhr bis 23 Uhr<br />
+          <i>So-Mo: Ruhetage</i></p>
+          <p>&nbsp;</p>
+          <h3>So erreichen Sie uns!</h3>
+          <p>Hotel & Restaurant Fiescherblick<br />
+            Dorfstrasse 203<br />
+            3818 Grindelwald<br />
+            Telefon: +41 12 123 12 12<br />
+          E-Mail: <a href="mailto:restaurant@hotel-fiescherblick.ch?subject=Reservation">restaurant@hotel-fiescherblick.ch</a></p>
+          <a class="myButton" href="mailto:restaurant@hotel-fiescherblick.ch?subject=Reservation">Tisch reservieren!</a>
+        </div>
+    </main>
+
+    <footer class="footer">
+      <div class="footer_menubar">
+          <div class="footer_menu_item">
```

Comment:
Minor thing but class name should be named in kebab-case, like this: `footer-menu-item`